BUILDING.md

```diff
@@ -0,0 +1,9 @@
+# Building
+
+This is how you can generate a .zip to upload to a WordPress instance.
+
+1. Run `composer install`.
```

Comment:
I would prefer something like
__You'll need the PHP extensions `mbstring`, `curl` and `simplexml`. If you're using Ubuntu, you can install these by running `sudo apt install php-mbstring php-curl php-simplexml`__ just to not pretend Ubuntu is the only operating system out there.

Reply:
Also, running `apt install` for a package that's already installed doesn't hurt, so there is not really a reason not to have them all in one line

Reply:
How can we check which extensions are required?